web/sablony/blackarrow/prihlaska.xtpl

```diff
@@ -195,7 +195,10 @@
   <div class="prihlaska_hlavicka">
     <h1>Přihláška na GameCon</h1>
 
-    <p>GameCon už proběhl, těšíme se na viděnou v roce {rok}.</p>
+    <p>
+      GameCon už proběhl, těšíme se na viděnou v roce {rok}. 
+      Můžeš se podívat, o co jsi přišel, na našem <a href="https://www.facebook.com/gamecon" title="Facebook Gameconu">Facebooku</a>.
```

Comment:
Hnidopišská - hodil bych to na nový řádek, nebo možná do vlastního paragrafu

```
<p>
      GameCon už proběhl, těšíme se na viděnou v roce {rok}. 
</p>
<p>
      Můžeš se podívat, o co jsi přišel, na našem <a href="https://www.facebook.com/gamecon" title="Facebook Gameconu">Facebooku</a>.
</p>
```

A nepřidáme i Instagram? https://www.instagram.com/gamecon_cz/?hl=en (nechám na tobě, já tam nechodím).

Reply:
Super, doplnil jsem.